src/store/reducers/nodes.ts

```diff
@@ -42,22 +57,25 @@ const nodes: Reducer<INodesState, INodesAction> = (state = initialState, action)
                 loading: false,
             };
         }
-        case CLEAR_NODES: {
+        case RESET_NODES_STATE: {
```

Comment:
data and totalNodes should be persistent when state resets?

Reply:
Yes. I will make state reset more explicit, so it will be clear which values are reset

Reply:
I didn't meant to set an explicit list of params. I just noticed that data and totalNodes are not reseted and wanted to clarify that it is correct...

Reply:
I think now it's better as it won't change, if some new values will be added to initial state. For example, default empty object for data, that will reset data on path change. Maybe I can set variables not as constants, but as `InitialState.loading` or `InitialState.searchValue` and etc.